pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/util/OncHistoryTestUtil.java

```diff
@@ -148,6 +150,17 @@ public Object createSmId(String guid, int tissueId, String realm, String userEma
         return patcher.doPatch();
     }
 
+    public Object createSmId(ParticipantDto participantDto, String eligibleSmId, DDPInstanceDto ddpInstanceDto) throws Exception {
```

Comment:
This should return a more specific type.  If we know it's going to be a `SMIDPatch`, have the method declare that as the return type, even if internally things need to be cast.